src/plugins/Webcam/index.js

```diff
@@ -168,7 +169,9 @@ module.exports = class Webcam extends Plugin {
       })
       return this.getVideo()
     }).then((file) => {
-      return this.uppy.addFile(file)
+      this.uppy.addFile(file)
```

Comment:
this no longer waits for the `addFile()` function to complete, is that intentional?

Reply:
No, just wanted to close the panel after file is added :) Will try to fix. But it could make sense to do this async, while the panel is being closed, the file is being added :)